### PR TITLE
PWGGA/GammaConv: add NDM pT vs HNM pT histo for eta prime, new cuts w…

### PR DESCRIPTION
**Pull request by EwaGlimos**
…ithout minimum pT

### CODE REVIEW

File: PWGGA/GammaConvBase/AliConversionMesonCuts.cxx [hunk 1]
```diff
@@ -1784,7 +1788,7 @@ Bool_t AliConversionMesonCuts::SetCut(cutIds cutID, const Int_t value) {
         UpdateCutString();
         return kTRUE;
       } else return kFALSE;
-    } else if(fUsePtDepSelectionWindow && fMesonKind == 2){
+    } else if(fUsePtDepSelectionWindow || fMesonKind == 3){
```
Comment (jokonig) [inline PWGGA/GammaConvBase/AliConversionMesonCuts.cxx:1791]: Why was this changed from && to || ?
Now everyone that has fUsePtDepSelectionWindow set to true will enter the eta prime case.

**Review by jokonig — CHANGES_REQUESTED** [review]

File: PWGGA/GammaConvBase/AliConversionMesonCuts.cxx (hunk 1 above)
Comment (EwaGlimos) [reply PWGGA/GammaConvBase/AliConversionMesonCuts.cxx:1791]: That's a very good point, I meant to delete the flag from there completely. Thank you!

**Comment by FriederikeBock** [comment]
+1
